Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1318,13 +1318,13 @@ void WebProcess::networkProcessConnectionClosed(NetworkProcessConnection* connec
     }
 
     for (auto& page : m_pageMap.values()) {
-        auto idbConnection = page->corePage()->optionalIDBConnection();
```

Comment:
`RefPtr corePage = page->corePage();` and then use this local variable below instead of calling `protectedCorePage()` twice.